clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4685,6 +4774,9 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     } else if (JA.getType() == types::TY_RewrittenLegacyObjC) {
       CmdArgs.push_back("-rewrite-objc");
       rewriteKind = RK_Fragile;
+    } else if (JA.getType() == types::TY_CMD) {
+      // DepScan response file output. Use fsyntax-only is enough.
+      CmdArgs.push_back("-fsyntax-only");
```

Comment:
Can you explain what this is for? I feel like the depscan job should be getting the "real" `-cc1` command, not something else.

Reply:
(If we need to specify *something*, seems like `-Eonly` might be a better fit, since it's only doing preprocessing in the end... but I'm wondering if we can avoid this entirely, or pass the options for the job-being-scanned-for?)

Reply:
This is the same question as above. Probably need JA to carry the correct final output type (would be good not to lose TY_CMD type as that is used to create temp file but that could be implicit).